cl/assets/react/_disclosure_helpers.tsx

```diff
@@ -1,256 +0,0 @@
-import { GROSS_VALUE, INCOME_GAIN, VALUATION_METHODS } from './_disclosure_models';
-import React from 'react';
-import { appFetch } from './_fetch';
-import debounce from 'lodash.debounce';
-
-interface Row {
-  id: number;
-  newest_disclosure_url: string;
-  name_full: string;
-  position_str: string;
-  disclosure_years: string;
-  thumbnail_path: string;
-}
-
-export function isDescendant(parent: HTMLElement | null, child: HTMLElement | null) {
-  if (child == null) {
-    return false;
-  }
-
-  let node = child.parentNode;
-  while (node != null) {
-    if (node == parent) {
-      return true;
-    }
-    node = node.parentNode;
-  }
-  return false;
-}
-
-export const fetchDisclosure = async (setData: any, parameters: any) => {
-  try {
-    const response: boolean | Query = await appFetch(
-      `/api/rest/v4/financial-disclosures/?person=${parameters['judge_id']}&id=${parameters['disclosure_id']}`
-    );
-    if (typeof response != 'boolean') {
-      const result = response['results'][0];
-      setData(result);
-    }
-  } catch (error) {
-    console.log(error);
-  }
-};
-
-export const convertTD = (value: string | number, table: string, key: string) => {
-  if (value == -1 || !value) {
```

Comment:
Do we still handle the -1 values properly?

Reply:
Yes\! The template tags in `cl/disclosures/templatetags/disclosure_tags.py` check for -1 values on lines 20, 35, 50, and 64. Each filter returns empty string for -1 values, and there's also an `is_failed_extraction` filter to show the eye-slash icon.